package.json

```diff
@@ -122,6 +122,7 @@
     "autoprefixer": "^10.4.2",
     "babel-plugin-import": "^1.13.3",
     "chai": "^4.3.4",
+    "dotenv-flow": "^3.2.0",
```

Comment:
I thought regular dotenv already worked well with multiple files? I think it goes in the order of
.env
.env.development
.env.local

The difference maybe being that variables can't be overwritten
A solution could be to just # out the variables in .env that needs to be overwritten by .local

Reply:
Yeah It kinda doesn't for the .env.local with the dotenv thing for the yaml configs